src/btree/bt_cursor.c

```diff
@@ -324,6 +321,14 @@ __wt_cursor_valid(WT_CURSOR_BTREE *cbt, WT_ITEM *key, uint64_t recno, bool *vali
         }
     }
 
+    if (btree->type != BTREE_ROW) {
```

Comment:
Add comment here to explain that we are checking against the update list now.

Reply:
0f4f0221d